qutip/qip/operations/gates.py

```diff
@@ -58,6 +59,114 @@
 #
 
 
+def x_gate(N=None, target=0):
+    """Pauli-X gate or sigmax operator.
+
+    Returns
+    -------
+    result : qobj
+        Quantum object for operator describing
+        a single-qubit rotation through pi radians around the x-axis.
+
+    Examples
+    --------
+    >>> sigmax()
```

Comment:
The example is not really useful here... I guess we can leave examples, they are pretty straightforward gate.

Reply:
Yeah, I already removed those.